zerver/openapi/zulip.yaml

```diff
@@ -11552,6 +11552,14 @@ paths:
                     - 4
                     - 5
                   example: 1
+                jump_to_sent_message_conversation:
+                  description: |
+                    Whether the user's view should automatically jump to the
+                    conversation where they sent a message.
+
+                    **Changes**: New in Zulip 9.0 (feature level 268).
```

Comment:
Add a sentence, "Previously, this behavior was not configurable."

Reply:
Done